### DISCUSSION TITLE
dokku network and postgres plugin

### DISCUSSION THREAD

**Discussion by mars3142** [dokku/dokku#5803] [Q&A]
After adding my app to a custom network, the app can't find the postgres db, which is installed as plugin. Any ideas how to fix that?

**Comment by josegonzalez** [comment]
Your likely set the `initial-network` and not `attach-post-create` or `attach-post-deploy`. The former removes the app from the bridge network, which is what is used by the postgres plugin by default.

You can either:

- switch the network setting at attach-post-create or attach-post-deploy (depends on your needs/why you changed the network)
- [set one of the network properties](https://github.com/dokku/dokku-postgres#set-or-clear-a-property-for-a-service) for your postgres database and then stop/start the database to have the network property apply
